devmode-quit.adoc

```diff
@@ -1,2 +1,2 @@
-When you are done checking out the service, exit development mode by typing `q` 
-in the shell session where you ran the server, and then press the `enter/return` key.
+When you are done checking out the service, exit development mode by pressing `CTRL+C` in the shell session
+where you ran the server, or by typing `q` and then pressing the `enter/return` key.
```

Comment:
IMO, we don't need to `q` option here, but I'm ok to include it. Looks good. ID review can review when reviewing the changes using this file.